src/transformers/models/patchtsmixer/configuration_patchtsmixer.py

```diff
@@ -0,0 +1,259 @@
+            The context/history length for the input sequence.
+        patch_len (`int`, *optional*, defaults to 8):
+            The patch length for the `PatchTSMixer` model. Try to set it as a divisor of `seq_len`.
+        input_size (`int`, *optional*, defaults to 1):
+            Number of input variates. For Univariate, set it to 1.
+        stride (`int`, *optional*, defaults to 8):
+            Determines the overlap between two consecutive patches. Set it to patch_len (or greater), if we want
+            non-overlapping patches.
+        num_features (`int`, *optional*, defaults to 8):
+            Hidden dimension of the model. Recommended to set it as a multiple of patch_len (i.e. 2-5X of patch_len).
+            Larger value indicates more complex model.
+        expansion_factor (`int`, *optional*, defaults to 2):
+            Expansion factor to use inside MLP. Recommended range is 2-5. Larger value indicates more complex model.
+        num_layers (`int`, *optional*, defaults to 3):
+            Number of layers to use. Recommended range is 3-15. Larger value indicates more complex model.
+        dropout (`float`, *optional*, defaults to 0.2):
+            The dropout probability the `PatchTSMixer` backbone. Recommended range is 0.2-0.7
+        mode (`str`, *optional*, defaults to `"common_channel"`):
+            Mixer Mode. Determines how to process the channels. Allowed values: "flatten", "common_channel",
+            "mix_channel". In "flatten" mode, patch embedding encodes the patch information across all channels. (not a
+            preferred approach) In "common_channel" mode, we follow Channel-independent modelling with no explicit
+            channel-mixing. Channel mixing happens in an implict manner via shared weights across channels. (preferred
+            first approach) In "mix_channel" mode, we follow explicit channel-mixing in addition to patch and feature
+            mixer. (preferred approach when channel correlations are very important to model)
+        gated_attn (`bool`, *optional*, defaults to `True`):
+            Enable Gated Attention.
+        norm_mlp (`str`, *optional*, defaults to `"LayerNorm"`):
+            Normalization layer (BatchNorm or LayerNorm).
+        self_attn (`bool`, *optional*, defaults to `False`):
+            Enable Tiny self attention across patches. This can be enabled when the output of Vanilla PatchTSMixer with
+            gated attention is not satisfactory. Enabling this leads to explicit pair-wise attention and modelling
+            across patches.
+        self_attn_heads (`int`, *optional*, defaults to 1):
+            Number of self-attention heads. Works only when `self_attn` is set to `True`.
+        use_pe (`bool`, *optional*, defaults to `False`):
+            Enable the use of positional embedding for the tiny self-attention layers. Works only when `self_attn` is
+            set to `True`.
+        pe (`str`, *optional*, defaults to `"zeros"`):
```

Comment:
```suggestion
        positional_encoding (`str`, *optional*, defaults to `"zeros"`):
```
I'd prefer to write it out here, `"pe"` is not that known IMO.

Reply:
https://github.com/huggingface/transformers/pull/26247/commits/49a485b35b85da165ec5b0f0ad95e8f500efcaec